tornadis/pubsub.py

```diff
@@ -20,6 +20,12 @@ class PubSubClient(Client):
     """
 
     def call(self, *args, **kwargs):
+        if args[0] == 'AUTH':
```

Comment:
I think it would be better to use "_call()" in the connect() method (instead on "call()").

This test would be useless